src/Spice86.Core/Emulator/OperatingSystem/IDosPathResolver.cs

```diff
@@ -0,0 +1,84 @@
+namespace Spice86.Core.Emulator.OperatingSystem;
+
+using Spice86.Core.Emulator.OperatingSystem.Structures;
+
+using System.Collections.Generic;
+
+/// <summary>
+/// Translates DOS filepaths to host file paths, and vice-versa.
+/// </summary>
+public interface IDosPathResolver {
+    /// <summary>
+    /// The map between host paths and the root of DOS drives.
+    /// </summary>
+    IDictionary<char, MountedFolder> DriveMap { get; }
+
+    /// <summary>
+    /// The current DOS drive in use.
+    /// </summary>
+    char CurrentDrive { get; }
+
+    /// <summary>
+    /// The full host path to the folder used by DOS as the current folder.
+    /// </summary>
+    string CurrentHostDirectory { get; }
```

Comment:
These three properties are never used outside the DosPathResolver itself. Is there a usecase for them to be accessible through the interface?

Reply:
I'd like to extend the debug window in the UI a lot, and show those properties, eventually.

Reply:
Properties removed.